packages/neovim/runtime-autoload-man.vim.patch

```diff
@@ -1,12 +0,0 @@
-diff -u -r ../neovim-0851057a8deaa1197bd0af22babb62c1146d836c/runtime/autoload/man.vim ./runtime/autoload/man.vim
---- ../neovim-0851057a8deaa1197bd0af22babb62c1146d836c/runtime/autoload/man.vim	2018-02-04 12:54:30.000000000 +0000
```

Comment:
This `.vim` file has been rewritten upstream using a lua script. So we should patch this script. See 
https://github.com/termux/termux-packages/blob/588653cc210ff6a63fc429cd7609d349aa82bdee/packages/neovim-nightly/runtime-lua-man.lua.patch.